src/main/java/seedu/stock/ui/HelpWindow.java

```diff
@@ -15,26 +16,143 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String HELP = "HELP \n---------------------------\n";
+    public static final String ADD = "---------------------------\nADD \n---------------------------\n";
+    public static final String DELETE = "---------------------------\nDELETE\n---------------------------\n";
+    public static final String FIND = "---------------------------\nFIND\n---------------------------\n";
+    public static final String UPDATE = "---------------------------\nUPDATE\n---------------------------\n";
+    public static final String EXIT = "---------------------------\nEXIT\n---------------------------\n";
+
+    public static final String HELP_DESCRIPTION =
+            "format: \nhelp\n\n";
+
+    public static final String ADD_DESCRIPTION =
+            "format: \nadd n/<name> s/<source of stock> q/<quantity> l/<location in warehouse>\n\n";
+
+    public static final String DELETE_DESCRIPTION =
+            "format: \ndelete sn/<serial number>\n\n";
+
+    public static final String FIND_DESCRIPTION =
+            "format(any): \nfind n/<name> find sn/<serial number> find l/<location> find s/<source of the stock>\n\n";
+
+    public static final String UPDATE_DESCRIPTION =
+            "format(multiple): \nupdate sn/<Serial Number of product> (followed by one of):\n"
+                    + "q/<+/-><quantity to increment/decrement> nq/<new quantity> n/<new name> "
```

Comment:
Based on our last meeting, I think prefix for quantity to increment/decrement should be changed to "iq/"?

Reply:
Noted! I will include the prefix for the quantity! I will make the necessary changes on the UG as well